src/index.ts

```diff
@@ -22,39 +22,44 @@ import { RunPayload, RunResult, RUN } from './RpcTypes';
 
 const checkerPluginName = 'fork-ts-checker-webpack-plugin';
 
-type Formatter = (message: NormalizedMessage, useColors: boolean) => string;
+namespace ForkTsCheckerWebpackPlugin {
```

Comment:
I hope these changes are okay? (see the whole file!)

Reply:
I think this will possibly be a breaking change for how types are consumed. Looking at code in the PR it looks like we switch from consuming as `Options` to `ForkTsCheckerWebpackPlugin.Options`.

Do I have that right? We'd need to do a new major version if so.

Reply:
These types weren't exported at all before, so tI think here could not exist anything out there consuming them.